patches/minecraft/net/minecraft/inventory/container/GrindstoneContainer.java.patch

```diff
@@ -1,5 +1,46 @@
 --- a/net/minecraft/inventory/container/GrindstoneContainer.java
 +++ b/net/minecraft/inventory/container/GrindstoneContainer.java
+@@ -27,6 +_,7 @@
+       }
+    };
+    private final IWorldPosCallable field_217015_e;
++   private final World level;
+ 
+    public GrindstoneContainer(int p_i50080_1_, PlayerInventory p_i50080_2_) {
+       this(p_i50080_1_, p_i50080_2_, IWorldPosCallable.field_221489_a);
+@@ -35,14 +_,15 @@
+    public GrindstoneContainer(int p_i50081_1_, PlayerInventory p_i50081_2_, final IWorldPosCallable p_i50081_3_) {
+       super(ContainerType.field_221521_o, p_i50081_1_);
+       this.field_217015_e = p_i50081_3_;
++      this.level = p_i50081_2_.field_70458_d.field_70170_p;
+       this.func_75146_a(new Slot(this.field_217014_d, 0, 49, 19) {
+          public boolean func_75214_a(ItemStack p_75214_1_) {
+-            return p_75214_1_.func_77984_f() || p_75214_1_.func_77973_b() == Items.field_151134_bR || p_75214_1_.func_77948_v();
++            return true;
+          }
+       });
+       this.func_75146_a(new Slot(this.field_217014_d, 1, 49, 40) {
+          public boolean func_75214_a(ItemStack p_75214_1_) {
+-            return p_75214_1_.func_77984_f() || p_75214_1_.func_77973_b() == Items.field_151134_bR || p_75214_1_.func_77948_v();
++            return true;
```

Comment:
Breaks vanilla behavior. This is my opinion, but you should check whether the stack is valid rather than just saying all stacks are valid. This could be justified in either direction however.